website/content/en/docs/helm/reference/advanced_features.md

```diff
@@ -91,7 +93,20 @@ Depending on the number of CRs of the same type, a single reconciling worker may
 For example:
 
 ```sh
-$ operator-sdk exec-entrypoint helm --max-workers 10
+$ cat config/manager/manager.yaml 
+...
+    spec:
+      containers:
+      - args:
+        - manager
+        - --max-workers=10
+...
+```
+
+Then, you will see this value when the container be initialized:
+
+```
+{"level":"info","ts":1593598237.085664,"logger":"controller-runtime.controller","msg":"Starting workers","controller":"nginx-controller","worker count":10}
```

Comment:
I think this logging is pretty obvious and doesn't need to be spelled out. It just makes this doc longer and more verbose with little extra value.